core/src/operations/decorators/debug.rs

```diff
@@ -13,13 +13,35 @@ pub enum DebugOptions {
     StackAll,
     /// Prints out the top n items of the stack for the current context.
     StackTop(u16),
+    /// Prints out the entire contents of RAM.
+    MemAll,
+    /// Prints out the contents of memory stored in the provided interval. Interval boundaries are
+    /// both inclusive.
+    MemInterval(u32, u32),
+    /// Prints out locals stored in the provided interval of the currently executing procedure.
+    /// Interval boundaries are both inclusive.
+    ///
+    /// Boolean parameter indicated whether whole local memory should be printed.
+    LocalInterval((u32, u32), u32, bool),
```

Comment:
Let's simplify this and define it as `LocalInterval(u32, u32, u32)` - i.e., we can get rid of the boolean flag. Basically, if the interval covers all locals we can print it out as such.

also, we should explain what the 3 values are in the comments above.

Reply:
But won't it be confusing that the user has requested values in a certain interval, and he is given a smaller interval without any boundaries? Probably, in this case, we need to additionally specify the total amount of local memory, so it should be clearer.